package.json

```diff
@@ -144,8 +148,10 @@
     "uuid": "2.0.1",
     "valid-url": "1.0.9",
     "walk": "2.3.4",
-    "webpack": "1.13.1",
-    "webpack-dev-middleware": "1.2.0",
+    "webpack": "3.0.0",
```

Comment:
I don't want to drag this PR any longer, so feel free to bump this version in another one. Webpack 3.1.0 has been released, and it solves the performance problems: https://github.com/webpack/webpack/releases/tag/v3.1.0